tests/js/searchtools.js

```diff
@@ -1,20 +1,20 @@
 describe('Basic html theme search', function() {
 
+  function loadFixture(name) {
+      req = new XMLHttpRequest();
+      req.open("GET", `base/tests/js/fixtures/${name}`, false);
+      req.send(null);
```

Comment:
I'm guessing you used `XMLHttpRequest()` because you want to make a synchronous? Should probably add a comment about that.

Apparently using [XMLHttpRequest this way is discouraged](https://stackoverflow.com/a/72561702/295132) and might be deprecated at some point, but they've been saying that since 2014. 😉 It's probably fine.

Reply:
Ok, that's good to know :)  Yep; we don't want the test to proceed until the fixture has definitely loaded -- I'm inclined to use a straightforward (even if seemingly dated) approach here.